### PR TITLE
feat(staff): add reviewers

### PR DESCRIPTION
**Pull request by CharisLai**
## Types of changes

* **New feature**

## Description
feat(staff): add reviewers

### CODE REVIEW

File: pages/about/staff.vue [hunk 1]
```diff
@@ -196,17 +196,48 @@ export default {
...
+                    { name: 'KK', photo: 'default.jpg' },
+                    { name: 'Chihchun Tsai', photo: 'default.jpg' },
+                    { name: '陳俊嘉 ccc', photo: 'default.jpg' },
+                ],
                 photography: [
                     { name: 'Andy', photo: 'photography/Andy.jpg' },
                     { name: 'Deimos', photo: 'photography/Deimos.jpg' },
-                    { name: 'Charles', photo: 'photography/Charles.jpg' },
-                    { name: '孝玠', photo: 'photography/孝玠.jpg' },
-                    { name: '庭蓁', photo: 'photography/庭蓁.jpg' },
```
Comment (wanyewei) [inline pages/about/staff.vue:205]: Why were items of photography deleted?

File: static/img/staff/program/kirchou.jpg
Comment (wanyewei) [inline static/img/staff/program/kirchou.jpg:1]: why is the program folder changed?

**Comment by wanyewei** [comment]
LGTM!

File: pages/about/staff.vue (hunk 1 above)
Comment (CharisLai) [reply pages/about/staff.vue]: Oh...it's a mistake, i already make it right. Thanks.

File: static/img/staff/program/kirchou.jpg
Comment (CharisLai) [reply static/img/staff/program/kirchou.jpg:1]: Ya...don't need to change this folder. i revise.

**Review by SivanYeh — APPROVED** [review]

**Comment by SivanYeh** [comment]
@wanyewei 讚讚! 下次記得最後按approve（approve 才會出現
image
）
：
image

**Pull request merged**